acceptance/tests/options/color.rb

```diff
@@ -0,0 +1,15 @@
+# This test is intended to ensure with --debug and --color, facter sends escape sequences to colorize the output
+test_name "C864545: --debug and --color command-line options should print DEBUG messages with color escape sequences" do
+  tag 'risk:high'
+
+  confine :except, :platform => 'windows' # On windows we don't get an escape sequence to detect to color change
+
+  agents.each do |agent|
+    step "Agent #{agent}: retrieve debug info from stderr using --debug and --color option" do
+      on(agent, facter('--debug --color'), :environment => { 'TERM' => 'xterm-256color' }) do |facter_output|
```

Comment:
I played around a bit more and it appears that we send the escape sequence regardless of what the terminal type setting is. (even dumb). I'm inclined to leave the terminal type set to give the test the best chance of working in CI just in case. I'll add a comment